lib/thermex/monitor.ex

```diff
@@ -31,29 +36,14 @@ defmodule Thermex.Monitor do
   end
 
   defp read_temperature(filename) do
-    lines = filename
-    |> File.stream!
-    |> Stream.map(&String.strip/1)
-    |> Enum.take(2)
-
-    {temperature, _} = parse_temperature(List.first(lines), List.last(lines))
-    temperature / 1000
-  end
-
-  defp parse_temperature(first_string, second_string) do
-    cond do
-      String.ends_with?(first_string, "YES") ->
-        String.split(second_string, "=")
-        |> List.last
-        |> Integer.parse
-      true ->
-        nil
-    end
+    filename
+    |> File.read!
```

Comment:
Why did you change this from `stream!` to `read!`? Just wondering. I know that read! will load the whole file into memory, but maybe the file is small enough that this doesn't matter.